packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -171,12 +171,12 @@ Nimble will introduce `nimble-table-column-number-text` which formats a numeric
-    -   `undefined` - use a default formatter, which will format similarly to [`Number.toString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toString#description). This displays integers with no trailing zeros, limits to about 16 significant digits, and switches to exponential notation for very large and small numbers. Instead of using `toString()` it will be implemented using `Intl.NumberFormat` to achieve more consistent i18n (`toString` always uses a `.` separator and displays the English word for "Infinity"). Will be displayed left-aligned by default (since numbers will display an inconsistent number of fractional digits).
-    -   `'integer'` - format all values as integers, rounding to nearest if the value isn't an integer and never displaying exponential notation. Will be displayed right-aligned by default.
+    -   `undefined` - use a default formatter that will display integers with no trailing zeros, limits to 6 digits, and exponential notation is used for numbers that are large (\`>= 1e6\`) or small (\`< 1e-3\`) in magnitude. Will be displayed left-aligned by default (since numbers will display an inconsistent number of fractional digits).
+    -   `'round-to-integer'` - format all values as integers, rounding to nearest if the value isn't an integer and never displaying exponential notation. Will be displayed right-aligned by default.
     -   `'decimal'` - format all values as decimal values (e.g. 123.45), always displaying `decimal-digits` digits after the separator and never displaying exponential notation. Will be displayed right-aligned by default.
     -   `'custom'` - use [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) configured with values specified in other attributes. Will be displayed left-aligned by default (since Nimble can't know if the provided configuration will produce consistent decimal separators).
     -   This could be extended to other pre-configured formats in future. Their configuration attributes would similarly be prefixed with the name of the format mode.
-    -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. For all modes besides `custom` Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887) (e.g. `useGrouping: false` to achieve `1000` rather than `1,000` and `signDisplay: auto` to achieve `1` rather than `+1`).
+    -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. For all modes besides `custom` Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887). The exception is that we will set `useGrouping: true` to achieve `1,000` rather than `1000` because this styles the values in a way that is more human readable.
```

Comment:
There was a bit of discussion around this, but it was just with the dev team. Something things we talked about were:
- commas will likely make the value more readable
- SLE doesn't have commas today, but SLS had commas. We don't think the change in behavior was intentional.

I agree that it would be good to extend the discussion to include the designers.

Reply:
Those are great points in favor of commas. I'm fine either way, just want to make sure we have some authority/rationale to point to when someone inevitably complains.

If designers aren't able to give guidance immediately then I'm ok proceeding with commas for now and revisiting this when we get firmer guidance. One idea though: why not use `useGrouping: auto` instead of `true`? The result is the same for `en-us` but the MDN docs imply it might make different decisions in other locales where grouping is less expected.

Reply:
Based on an offline discussion with the dev team and designers, we decided to render values with commas.

Regarding using `useGrouping: auto`, that [isn't supported in FF yet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#browser_compatibility). In FF, values other than `boolean | undefined` are in preview mode.

Reply:
Seems like once widely available in ffx (release + 6 months) we should switch to that as a better behavior and clean-up some of the workarounds for other missing options, could that be captured in a tech debt?

Reply:
Created #1442